### DISCUSSION TITLE
Document edit

### DISCUSSION THREAD

**Discussion by mandarinSh** [hcengineering/platform#4897] [Q&A] [answered]
Hey guys, how to add tables to documents and edit them like remove columns or so?
Thanks!

**Comment by aplatoff** [comment, marked as answer]
Hello,

Thank you for asking, it's not convenient now: you have to select something in order to popup `format` menu, and click table icon there:

<img width="514" alt="Screenshot 2024-03-12 at 19 49 48" src="https://github.com/hcengineering/platform/assets/87076238/fab3e450-040b-41bb-88cd-fee62cbb4333">

Once you create table, adding removing columns looks to be straightforward:

<img width="844" alt="Screenshot 2024-03-12 at 19 50 02" src="https://github.com/hcengineering/platform/assets/87076238/a4fe7939-865c-42ee-88ad-0d82c341abb3">
